designs/administrator/olm/installing-operator-long-running-component/index.md

```diff
@@ -0,0 +1,57 @@
+---
+parent: Administrator
+version: 4.7
+---
+
+# Operator Installing with Long Running Component
+
+Shortly after the new screen to Convey an operator is installing was developed, it was realized that other actions in the console could make use of a similar interaction. This design conveys a more generic version of the operator installing screen, and examples of perhaps how other actions could use a similar interaction.
+
+## Operator Installation
+
+### Installing an operator
+![Install operator](img/longRun-1-1-installing.png)
```

Comment:
Hi, this might be application specific but in OpenShift Console, operator details page looks like:
![image](https://user-images.githubusercontent.com/12733153/90021621-847d8f80-dc7f-11ea-9d6b-a6a98f09a353.png)

Note the breadcrumb.  If the 'long running' contents are to appear under the breadcrumbs, there might not be a need for the line "View Installed Operators in namespace..."  -at minimum the link should be configurable to hide or show.